builtin/providers/google/config.go

```diff
@@ -7,9 +7,12 @@ import (
 	"net/http"
 	"os"
 
-	"code.google.com/p/goauth2/oauth"
-	"code.google.com/p/goauth2/oauth/jwt"
 	"code.google.com/p/google-api-go-client/compute/v1"
+	// oauth2 "github.com/rasa/oauth2-fork-b3f9a68"
+	"github.com/rasa/oauth2-fork-b3f9a68"
+
+	// oauth2 "github.com/rasa/oauth2-fork-b3f9a68/google"
+	"github.com/rasa/oauth2-fork-b3f9a68/google"
```

Comment:
Can we verify that this fork is still necessary? I found https://github.com/mitchellh/packer/commit/10dee1bd686e7443f7c4aca90cfcbfad970bf0f9 but I can't seem to find any commentary on what exactly broke and what needs to be fixed upstream.